### ISSUE TITLE
get_image() with GGUI

### ISSUE THREAD

**Issue by zhouxian** [taichi-dev/taichi#5263]
It seems `window.get_image()` is not available when using GGUI. Is there anyway to get the image in the form of an numpy array?

**Comment by neozhaoliang** [comment]
@zhouxian  I think `write_image` is available in ggui? Could you post a code snippet?

**Comment by zhouxian** [comment]
@neozhaoliang Yeah but that writes an image to the disk which is not very efficient. Is there any way to get an numpy array image, as available with the 2D GUI?

**Comment by neozhaoliang** [comment]
I think there is no direct way to do this for now.

**Comment by neozhaoliang** [comment]
This should be a desired feature, I'm reopening this and adding it to the todo list.
